### ISSUE TITLE
About the spatial_head in openpsg/models/roi_extractors/visiual_spatial.py

### ISSUE THREAD

**Issue by shuizaola** [Jingkang50/OpenPSG#64]
I am very interested in your work.I would really appreciate it if you could help me answer this question.
in line 516： roi_feats_result.append(head((roi_feats + rect_feats).view( roi_feats.size(0), -1))) ，I know that roi_feats represent the features of the union of subject and object, so what does rect_feats represent?
in line 474 :rect_input = torch.stack((head_rect, tail_rect),dim=1) 
in line 477: rect_feats = self.spatial_conv(rect_input)   What features does the rect_feats represent?（Does it represent the spatial featuresof subject and predicate? So what does this spatial feature mean?）

**Comment by Jingkang50** [comment]
Thanks for asking. If I remember it correctly, the spatial feature encodes the object location info by marking the localization of objects in interests in the original image with rect and generate feature accordingly. The code is from MMSceneGraph, so we just use it. You might need to print and check `rect_input` to explore / learn on your own.

**Comment by shuizaola** [comment]
> Thanks for asking. If I remember it correctly, the spatial feature encodes the object location info by marking the localization of objects in interests in the original image with a rectangular bbox coordinate (or mask) and generate feature accordingly. The code is from MMSceneGraph, so we just use it. You might need to print and check to explore / learn on your own.`rect_input`

Thanks you！
